Models/Pieces/Piece.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Chess_Backend.Models.Positions;
+
+namespace Chess_Backend.Models.Pieces
+{
+    public abstract class Piece
+    {
+
+        public abstract Color Color { get; }
```

Comment:
The color field and Tile field shouldn't be abstract, these field will hold info and will be the same across all the concrete class(those class which will extend Piece